### PR TITLE
Update the schedule for the coach meetings.

### PR DESCRIPTION
**Pull request by verwoerd**

### CODE REVIEW

File: content/schedule/coach_meeting_bapc.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+day: sunday
+start: 12:15
+duration: 1h15m
+width: 1
+location: Aula - Lecture Hall D
+people: coaches from Benelux area
+---
+## Location
+The BAPC coach meeting takes place in the Auditorium of the [Aula Conference Centre](https://iamap.tudelft.nl/en/poi/aula-conference-center/).
+The entrance of the hall D is on the top floor, and indicated on the signs.
```
Comment (robertdijk) [inline content/schedule/coach_meeting_bapc.md:13]:
```suggestion
The entrance of the hall D is indicated on the signs.
```

File: content/schedule/coach_meeting_gcpc.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+day: sunday
+start: 11:00
+duration: 1h15m
+width: 1
+location: Aula - Lecture Hall D
+people: German coaches
+---
+## Location
+The NCPC coach meeting takes place in the Auditorium of the [Aula Conference Centre](https://iamap.tudelft.nl/en/poi/aula-conference-center/).
+The entrance of the hall D is on the top floor, and indicated on the signs.
```
Comment (robertdijk) [inline content/schedule/coach_meeting_gcpc.md:13]:
```suggestion
The entrance of the hall D is indicated on the signs.
```

File: content/schedule/coach_meeting_ncpc.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+day: sunday
+start: 12:15
+duration: 1h15m
+width: 1
+location: Aula - Lecture Hall C
+people: Nordic and Baltic coaches
+---
+## Location
+The NCPC coach meeting takes place in the Auditorium of the [Aula Conference Centre](https://iamap.tudelft.nl/en/poi/aula-conference-center/).
+The entrance of the hall C is on the top floor, and indicated on the signs.
```
Comment (robertdijk) [inline content/schedule/coach_meeting_ncpc.md:13]:
```suggestion
The entrance of the hall C is indicated on the signs.
```

File: content/schedule/coach_meeting_ukiepc.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+day: sunday
+start: 11:00
+duration: 1h15m
+width: 1
+location: Aula - Lecture Hall C
+people: UK and Ireland coaches
+---
+## Location
+The UKIEPC coach meeting takes place in the Auditorium of the [Aula Conference Centre](https://iamap.tudelft.nl/en/poi/aula-conference-center/).
+The entrance of the hall D is on the top floor, and indicated on the signs.
```
Comment (robertdijk) [inline content/schedule/coach_meeting_ukiepc.md:13]:
```suggestion
The entrance of the hall D is indicated on the signs.
```

**Review by robertdijk — CHANGES_REQUESTED** [review]
I'm not sure if we'll use those entrances, or the entrances one floor down.

**Review by robertdijk — APPROVED** [review]

**Review by mpsijm — APPROVED** [review]
Fixed a few small things, you can merge if you agree :smile:

**Pull request merged**